test/handlers/admin/data-contributions.js

```diff
@@ -83,10 +83,17 @@ describe('admin data contributions handler', () => {
     it('updates the DataContribution and redirects to /admin/data-contributions', () => {
       const payload = {
         approved: true,
+        trial_id: '11111111-1111-1111-1111-111111111111',
+        data_category_id: null,
         curation_comments: 'My comments',
       };
       let dataContributionId;
 
+      factory.create('dataCategory')
+        .then((dataCategory) => {
+          payload.data_category_id = dataCategory.id;
+        });
```

Comment:
Can we discard this for now? It has been removed by #691 just recently, so it's going to die anyway.